classes/genre.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'music_album'
+
+class Genre
+  attr_accessor :name
```

Comment:
attr_accessor should be for items not for name you can see further description from the UML diagram